media/js/firefox/family/fx-is-default.es6.js

```diff
@@ -22,18 +22,23 @@ const FirefoxDefault = {
     },
 
     init: () => {
+        const main = document.querySelector('main');
+
         if (!FirefoxDefault.isSupported()) {
             return;
         }
-        return new window.Promise(function (resolve) {
-            FirefoxDefault.isDefaultBrowser()
-                .then(function () {
-                    document
-                        .querySelector('main')
-                        .classList.add('is-firefox-default');
-                    resolve();
-                })
-                .catch(() => resolve());
+
+        Mozilla.UITour.ping(() => {
+            main.classList.add('set-default-supported');
```

Comment:
This fixes a bug in the current CTD page where the "Set Firefox as default" CTA was never being shown. It's worth testing on both Firefox / non-Firefox browsers that the right CTAs are being displayed:

1. Non-Firefox browsers should see a "Download Firefox" CTA pointing to `/firefox/download/thanks/`.
2. Firefox desktop browsers should see a "Set as Default" CTA if not default, pointing to `/firefox/set-as-default/thanks/`
3. Firefox desktop browsers should see a "Firefox mobile" link if default, pointing to `/firefox/browsers/mobile/`